flake.nix

```diff
@@ -73,6 +73,11 @@
             );
           };
 
+        devShells.slim = with pkgs.ocamlPackages; pkgs.mkShell {
+          inputsFrom = [ dune_3 ];
+          buildInputs = [ merlin ];
```

Comment:
Wouldn't this be a bit more useful if you included some of the deps needed to run the tests? At least ppx_expect for example.

Reply:
good point. added some of the test inputs that gets `make test` further